api/public.proto

```diff
@@ -794,10 +874,78 @@ message Trip {
   }
 }
 
+// The Vehicle resource.
+//
+// This resource corresponds to the [vehicle position type in the GTFS static
+// specification](https://developers.google.com/transit/gtfs-realtime/reference#message-vehicleposition).
 message Vehicle {
+  string id = 1;
+
+  optional Trip.Reference trip = 2;
+
+  // Positional info
+  optional double latitude = 3;
+  optional double longitude = 4;
+  optional float bearing = 5;
+  optional double odometer = 6;
+  optional float speed = 7;
```

Comment:
Just as an FYI, doc strings on the proto fields ultimately get converted into a table in the documentation. [Here's the table for the route resource](https://docs.transiter.dev/api/public_resources/#route). So in general it's nice to have doc strings on every field. No pressure though, I can do it.

The docs are currently a mess - aside from the API reference, they haven't been updated since I did the Python -> Go migration. I'm actually working right now on rewriting them. So right now having nice docs is not a big deal.

Reply:
I can definitely add some more comments here.